zwave_js_server/model/log_config.py

```diff
@@ -42,9 +42,9 @@ def to_dict(self) -> LogConfigDataType:
     def from_dict(data: LogConfigDataType) -> "LogConfig":
         """Return LogConfig from LogConfigDataType dict."""
         return LogConfig(
-            data.get("enabled"),
-            data.get("level"),
-            data.get("logToFile"),
-            data.get("filename"),
-            data.get("forceConsole"),
+            data["enabled"],
```

Comment:
Are all keys required now? Then we can update `LogConfigDataType` to have total  True.

Reply:
I don't think they ever were optional

Reply:
Actually, in the reverse (when we update the log config), we support sending partial parameters, so I don't think we can set `Total` to `False`